lfr/7pin_cardinal_demo/HC.i

```diff
@@ -0,0 +1,477 @@
+half_pinpitch = 0.0067123105    # m
+fuel_r_o = 0.004318648          # m
+fuel_r_i = 0.00202042           # m
+gap_i = ${fparse fuel_r_i * 0.25}
+clad_r = 0.0054037675           # m
+duct_thickness = 0.0035327      # m
+flat_to_flat = 0.037164         # m
+activefuelheight = 1.06072      # m
+lowerreflectorheight = 0.2      # m
+upperreflectorheight = 0.2      # m
+
+num_coolreg = 8
+num_coolreg_back = 8
+num_layers_fuel = 40
+num_layers_refl = 8
+numside = 6
+
+linearpower = 27466.11572112955 # W/m
+inlet_T = 693.15                # K
+#fuelconductance = 1.882         # W/m/K
+cladconductance = 21.6          # W/m/K
+gapconductance = 0.251          # W/m/K
+
+bid_gapc = 9
+bid_gap = 1
+bid_fl  = 100
+bid_clad = 2
+bid_cool = 4
+bid_duct = 3
+bid_lrfl = 5
+bid_urfl = 6
+bid_lrflc = 7
+bid_urflc = 8
+
+half_asmpitch = ${fparse flat_to_flat / 2 + duct_thickness}
+powerdensity = ${fparse linearpower / (pi * (fuel_r_o * fuel_r_o - fuel_r_i * fuel_r_i))}
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = 'HCmesh_out.e'
+    exodus_extra_element_integers = 'pin_id'
+  []
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  l_max_its = 50
+  l_tol = 1e-9
+  nl_max_its = 50
+  nl_rel_tol = 1e-6
+  nl_abs_tol = 1e-8
+
+
+  dt = 0.0455311973018550    # 0.9106239460371001 ms (deltaT_NekRS) * 50
```

Comment:
I d probably recommend a 'common.i' input that you use to synch up these things between codes

Reply:
I will pass the comment to @pak1sol for consideration.

Reply:
Good idea. Let me work with @hapfang to get this done.

Reply:
I have a question while I'm at this. So I factored out common parameters in NT.i and HC.i to common.i for example and I ran `cardinal-opt -i common.i NT.i`. But the code complains about HC.i which is an input of the sub-app of NT.i not having parameters needed which are moved to common.i. In this multiapp setting, this doesn't seem to work. Am I missing something or is it something MOOSE needs to be updated?

Reply:
You ll want to use the WASP include instead of this. The multiapp syntax does not support this

Reply:
@GiudGiud OK. Thanks!

Reply:
@GiudGiud @pak1sol It looks like the include at the beginning of input files are causing errors for VTB Documentation. 
https://github.com/hapfang/virtual_test_bed/blob/4e9d3bf9d38b55247d4981b968c28fff49dd3d0b/lfr/7pin_cardinal_demo/NT.i#L1-L3
```
MooseDocs.base.readers (ForkProcess-9):
TOKENIZE ERROR: /home/fangj/projects/virtual_test_bed/lfr/7pin_cardinal_demo/NT.i:1: invalid character '!' - did you leave a field value blank after a previous '='?
```
Is there a way that we can avoid this error? Thanks.